bindata/network/host-network/ns.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  # NOTE: this namespace is the portal for host network traffic
+  name: openshift-host-network
+  labels:
+    network.openshift.io/policy-group/host-network: "true"
```

Comment:
Is that even accepted as a label name? It's supposed to be `[<prefix>/]<name>` where neither the `<prefix>` nor the `<name>` can have a slash in it. So, `policy-group.network.openshift.io/host-network` (and `""`), as Casey originally suggested.

We could suggest that people could write the policy as:

    - namespaceSelector:
        matchExpressions:
          - key: policy-group.network.openshift.io/host-network
            operator: Exists

if they find the `""` confusing...

OK, last chance to bikeshed! Is "policy-group" still the best phrase? (Does using the `matchExpression` syntax change that?) We need "`network.openshift.io`" in the prefix, so saying "`networkpolicy`" rather than just "`policy`" feels unnecessary. Do we want to drop "`group`"? Should the ingress label use the word "router[s]" rather than "ingress" to avoid confusion with the NP meaning of "ingress"? (Probably!)

Reply:
You are right, that syntax is not valid: https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/#syntax-and-character-set. Rolling back to our original label format. `"policy-group.network.openshift.io/ingress": ""` seems most prudent. There are more confusing things in the pipeline when router gets deprecated by ingressv2 :). Might as well keep ingress.